content/en/firewall.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Firewall
+status: Feedback Appreciated
+category: Technology
+---
+
+## What it is
+A firewall is a system that filters network traffic to protect against malware and network-based threats. It can either let traffic pass or block it based on specified rules. Firewalls can be hardware, software, or a combination of the two.
+
+## Problem it addresses
+Without a firewall, any traffic could potentially enter a network and interfere. Take a [microservices](/microservices/)-based banking app. The services are sending highly sensitive financial data over their network to communicate with one another. With no firewall, a malicious actor could enter the network, intercept communication, and inflict damage.
```

Comment:
Please rewrite this section to use the [active voice](https://www.grammar-monster.com/glossary/active_voice.htm). Also lets avoid mentioning a firewall directly in this section. Something like:

By default, a given network will allow all enter and leave it's boundaries as long as it conforms to the network's routing rules. This default behavior means that it is difficult to secure a given network space.